processor/src/main/java/org/mobilitydata/gtfsvalidator/processor/MixedCaseValidatorGenerator.java

```diff
@@ -70,11 +70,14 @@ private static TypeSpec generateValidator(GtfsFileDescriptor fileDescriptor) {
       if (!mixedCaseField.mixedCase()) {
         continue;
       }
+      String test = "blah";
+
       validateMethod
           .beginControlFlow(
               "if (entity.$L())", FieldNameConverter.hasMethodName(mixedCaseField.name()))
           .addStatement("$T value = entity.$L()", String.class, mixedCaseField.name())
-          .beginControlFlow("if (!(value.matches(\".*[a-z].*\") && value.matches(\".*[A-Z].*\")))")
+          .beginControlFlow(
+              "if (value.matches(\".*\\\\p{L}{2}.*\") && (value.toLowerCase() == value || value.toUpperCase() == value))")
```

Comment:
`\\p{L}{2}` matches any kind of letter from any language, including accented letters and special characters like ç and é.  With this logic, any value with 2 or more consecutive letters will be checked to see if it is all upper or lower case, and add a warning notice if so.  

**Question:** is "2 or more" adequate? The original issue #1402 also had an example value of `ZA12`.  Do we want to try to accommodate values with multiple number/letter combinations or is this adequate since this a warning and not an error?

/cc @isabelle-dr